### ISSUE TITLE
Cannot boot with TuxOnIce 4.4.0-22

### ISSUE THREAD

**Issue by ysalmon** [NigelCunningham/tuxonice-kernel-old#15]
Hello,
recently, I have not been able to boot my computer with 4.4.0-22-generic-tuxonice. It worked in the past (with -21, and I think once with -22 but not sure) and I do not understand what is going on.
The boot process stalls just after the following is displayed

```
mai 12 19:19:14 yann-Precision-T1600 kernel: Hardware name: Dell Inc. Precision T1600/06NWYK, BIOS A16 09/24/2013
mai 12 19:19:14 yann-Precision-T1600 kernel:  0000000000000286 00000000e28f7846 ffff88003556fd60 ffffffff813fed33
mai 12 19:19:14 yann-Precision-T1600 kernel:  0000000000000026 0000000000001000 ffff88003556fd80 ffffffff810cd38b
mai 12 19:19:14 yann-Precision-T1600 kernel:  ffff880414c71000 0000000000000026 ffff88003556fda0 ffffffff810cd6e3
mai 12 19:19:14 yann-Precision-T1600 kernel: Call Trace:
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff813fed33>] dump_stack+0x63/0x90
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810cd38b>] free_update_stats+0x4b/0x80
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810cd6e3>] toi_free_page+0x43/0x50
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810dc5c2>] forget_signature_page+0x42/0xd0
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810d98b3>] toi_bio_cleanup+0x33/0x70
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810cced1>] toi_cleanup_modules+0x41/0x60
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810ce9db>] toi_finish_anything+0x1b/0x90
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff810cd9e7>] toi_attr_store+0x127/0x2d0
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff812a08a7>] sysfs_kf_write+0x37/0x40
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff8129fcbd>] kernfs_fop_write+0x11d/0x170
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff81220338>] __vfs_write+0x18/0x40
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff81220cc9>] vfs_write+0xa9/0x1a0
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff81221985>] SyS_write+0x55/0xc0
mai 12 19:19:14 yann-Precision-T1600 kernel:  [<ffffffff8183a3b2>] entry_SYSCALL_64_fastpath+0x16/0x71
mai 12 19:19:14 yann-Precision-T1600 kernel: Idx  Allocs   Frees   Tests   Fails     Max Description
mai 12 19:19:14 yann-Precision-T1600 kernel:  38       0       1       0       0       0 setting swap signature
```

These lines remain on screen and the system remains responsive : I can restart the computer with Ctrl+Alt+Del.

I recently reformated my swap partition, which changed its UUID, but I did update /etc/fstab.

**Comment by mschlaeffer** [comment]
Which version of 4.4.0-22-generic-tuxonice are you using?

**Comment by ysalmon** [comment]
4.4.0-22.39~ppa1 amd64

**Comment by ysalmon** [comment]
Problem solved in 4.4.0-22.40~ppa1 ; thanks a lot !
